presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue when executing spatial join on top of a right join. See :issue:`12672`.
+* Fix an issue where ``TRY`` cannot catch the failures when ``ROW`` and ``ARRAY`` with ``NULLs`` fails to compare.
+* Lazily build the hash table for map.
```

Comment:
@yingsu00 Could you provide more context here. For example, does it improve performance in certain cases?

Reply:
Something like improve performance in scan which contains maps

Reply:
@tdcmeehan thanks for the explanation! Presto builds hashtable for MapBlocks eagerly when constructing the MapBlock even it's not needed in the query. Building a hashtable could take up to 30-40% CPU of the scan cost on a map column. This commit defers the hashtable build to the time it's needed in SeekKey(). So I would rephrase the release note as:
```
Lazily build the hash tables for MapBlock until it's probed, or skip building the hash tables for queries that don't need to probe them.
```